config/.env.dev.txt

```
@@ -20,6 +20,7 @@ NOTION_SECRET=this_is_not_the_secret
 GOOGLE_MAPS_API_KEY=this_is_not_the_secret
 ERP_API_KEY=this_is_not_the_secret
 ERP_SECRET=this_is_not_the_secret
+ERP_ENDPOINT=https://your-erp-host/api/resource
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ERP_ENDPOINT key should go before the ERP_SECRET key

(UnorderedKey)